### PR TITLE
Added the --sudo argument

### PR DESCRIPTION
**Pull request by Taliik**
## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -5,6 +5,9 @@ Changes for croud
 Unreleased
 ==========
 
+- Added the optional ``--sudo`` argument to be able to specify if a command should
```
Comment (MarkusH) [inline CHANGES.rst]:
```suggestion
- Added the optional ``--sudo`` argument to be able to specify when a command
  should
```

File: tests/unit_tests/test_rest.py [hunk 2]
```diff
@@ -47,23 +47,29 @@ def test_send_success_sets_data_with_key(self, mock_cloud_url, mock_token):
         with mock.patch.object(HttpSession, "_get_conn", return_value=self._get_conn()):
             data = {"key": "value"}
 
-            client = Client(env="dev", region="bregenz.a1", output_fmt="json")
+            client = Client(
+                env="dev", region="bregenz.a1", output_fmt="json", sudo=False
```
Comment (MarkusH) [inline tests/unit_tests/test_rest.py]: No need to pass `sudo=False` here all the time.

File: croud/rest.py [hunk 3]
```diff
@@ -34,20 +34,24 @@ class Client:
...
     _token: str
     _region: str
+    _sudo: bool
     _output_fmt: str
     _data: Optional[Union[List[dict], dict]] = None
     _error: Optional[dict] = None
 
-    def __init__(self, env: str, region: str, output_fmt: str, loop=None):
+    def __init__(
+        self, env: str, region: str, output_fmt: str, sudo: bool = False, loop=None
```
Comment (MarkusH) [inline croud/rest.py]: Let's add `sudo` at the end to keep the argument order. Or make `loop` and `sudo` keywork only arguments:
```python
def __init__(
    self, env: str, region: str, output_fmt: str, loop=None, sudo: bool = False
):
```
```python
def __init__(
    self, env: str, region: str, output_fmt: str, *, loop=None, sudo: bool = False
):
```

**Comment by Taliik** [comment]
> I think the former was correct. Now you're sending `X-Auth-Sudo: 0` which is an invalid header value.

So i should just check that the header is not there at all when sudo=False i suppose

**Pull request merged**